src/multi_vector_simulator/D0_modelling_and_optimization.py

```diff
@@ -284,15 +288,39 @@ def simulating(dict_values, model, local_energy_system):
         -------
         Updated model with results, main results (flows, assets) and meta results (simulation)
         """
+        import warnings
+
         logging.info("Starting simulation.")
-        local_energy_system.solve(
-            solver="cbc",
-            solve_kwargs={
-                "tee": False
-            },  # if tee_switch is true solver messages will be displayed
-            cmdline_options={"ratioGap": str(0.03)},
-        )  # ratioGap allowedGap mipgap
-        logging.info("Problem solved.")
+        # turn warnings into errors
+        warnings.filterwarnings("error")
+        try:
+            local_energy_system.solve(
+                solver="cbc",
+                solve_kwargs={
+                    "tee": False
+                },  # if tee_switch is true solver messages will be displayed
+                cmdline_options={"ratioGap": str(0.03)},
+            )  # ratioGap allowedGap mipgap
+        except UserWarning as e:
+            error_message = str(e)
+            compare_message = "termination condition infeasible"
+            if compare_message in error_message:
+                error_message = (
+                    f"The following error occurred during the mvs solver: {error_message}\n\n "
+                    f"There are several reasons why this could have happened."
+                    "\n\t- the energy system is not properly connected. "
+                    "\n\t- the capacity of some assets might not have been optimized. "
+                    "\n\t- the demands might not be supplied with the installed capacities in "
+                    "current energy system. Check your maximum power demand and if your energy "
+                    "production assets and/or energy conversion assets have enough capacity to "
+                    "meet the total demand"
```

Comment:
Another error category should be "uninitialized Numeric...".

Reply:
Ok! Just make sure to leave the issues open that remind of of that ;)

Reply:
I would close the issue for now and it can be reopen when needed (it remains searchable, having it still open suggests that nothing has been done to fix it)